packages/web-react/src/components/Card/demo/CardGeneralOptions.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Button } from '../../Button';
+import { Grid } from '../../Grid';
+import { Link } from '../../Link';
+import Card from '../Card';
+import CardBody from '../CardBody';
+import CardEyebrow from '../CardEyebrow';
+import CardFooter from '../CardFooter';
+import CardLink from '../CardLink';
+import CardMedia from '../CardMedia';
+import CardTitle from '../CardTitle';
+import { MEDIA_IMAGE } from './constants';
+
+const CardGeneralOptions = () => (
+  <Grid cols={{ mobile: 1, tablet: 2, desktop: 3 }}>
+    <Card>
+      <CardMedia>
+        <img src={MEDIA_IMAGE} alt="" />
+      </CardMedia>
+      <CardBody>
+        <CardEyebrow>Eyebrow title</CardEyebrow>
+        <CardTitle isHeading>
+          <CardLink href="#">Basic card</CardLink>
+        </CardTitle>
+        {/* User content */}
+        <p>
+          Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean fermentum risus id tortor. Integer lacinia.
+          Sed vel lectus.
+        </p>
+        <Link href="#" elementType="div" underlined="always">
```

Comment:
this shouldn't be a <Link />. 

```suggestion
<div class="link-primary link-underlined">Read more</div>
```